TYPO3.Flow/Classes/TYPO3/Flow/Utility/PositionalArraySorter.php

```diff
@@ -247,7 +247,7 @@ protected function extractAfterKeys(array &$arrayKeysWithPosition)
     {
         $this->afterKeys = array();
         foreach ($arrayKeysWithPosition as $key => $position) {
-            if (preg_match('/^after ([a-zA-Z0-9]+)(?: ([0-9]+))?$/', $position, $matches) < 1) {
+            if (preg_match('/^after (\S+)(?: ([0-9]+))?$/', $position, $matches) < 1) {
```

Comment:
it allows positional keys to contain special characters (such as "." when referring to package keys). One could consider it a bugfix, I'll push it to a separate PR